src/SprykerEco/Zed/Payone/Business/PayoneBusinessFactory.php

```diff
@@ -357,6 +359,16 @@ protected function createGenericPayment(Store $store)
         return $genericPayment;
     }
 
+    /**
+     * @param \Spryker\Shared\Kernel\Store $store
+     *
+     * @return \SprykerEco\Zed\Payone\Business\Payment\MethodMapper\Klarna
+     */
+    public function createKlarna(Store $store): Klarna
```

Comment:
Maybe we need to add KlarnaInterface to implement it in Klarna and return here

Reply:
++